src/pyseekdb/__init__.py

```
@@ -60,10 +60,11 @@
 
 import importlib.metadata
 
-# Note: pylibseekdb built with ABI=0 and onnxruntime built with ABI=1, so there's a conflict between the two libraries.
+# Note: pylibseekdb built with ABI=0 and torch built with ABI=1, so there's a conflict between the two libraries.
 # pylibseekdb is built both with ABI=0 and the -Bsymbolic flag, so we can load libraries with ABI=1 first
 # and then pylibseekdb to avoid these conflicts.
-import onnxruntime  # noqa: F401
+if importlib.util.find_spec("torch"):
+    import torch  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)